analytics_data_api/v0/documents.py

```diff
@@ -160,6 +160,8 @@ def get_users_in_course(
             }
             for sort_policy in sort_policies
         ])
+        total = search_request.count()
+        search_request = search_request[0:total]
```

Comment:
I'd have a new variable called `all_search_requests` or `total_search_requests`. I just don't like re-purposing same variable this way.

Reply:
this is actually how elasticsearch_dsl wants you to work!